.circleci/config.yml

```diff
@@ -951,6 +952,46 @@ jobs:
           artifact_name: LTE AGW
           version_path: versions/magma_version
 
+  lte-agw-aws:
+    environment:
+      #AWS_ACCESS_KEY_ID: "$AWS_FB_CRED"
+      #PARAM_AWS_SECRET_ACCESS_KEY: "$(printenv AWS_FB_SECRET)"
+      MAGMA_ROOT: /home/circleci/project  
+    machine:
+      image: ubuntu-1604:201903-01
+    steps:
+      - checkout
+      - python/set_version
+      - run:
+          command: |
+            pip install wheel
+            pip install ansible 
+            pip install awscli
+            pip install boto3
+            echo export AWS_DEFAULT_REGION="us-west-1" >> $BASH_ENV
+            echo export AWS_ACCESS_KEY_ID="$(printenv AWS_FB_CRED)" >> $BASH_ENV
+            echo export AWS_SECRET_ACCESS_KEY="$(printenv AWS_FB_SECRET)" >> $BASH_ENV
+            mkdir ~/magma-experimental
+            mkdir ~/magma-experimental/files
+            cd ${MAGMA_ROOT}/experimental/cloudstrapper/playbooks
+            cp -r * ~/magma-experimental
+            cp ~/magma-experimental/roles/vars/secrets.yaml ~/magma-experimental/files
+            sed -i -e "s/AWS_ACCESS_KEY_ID/${AWS_FB_CRED}/g" ~/magma-experimental/files/secrets.yaml
+            sed -i -e "s@AWS_SECRET_ACCESS_KEY@${AWS_FB_SECRET}@g" ~/magma-experimental/files/secrets.yaml
+            sed -i -e "s@imgCloudstrapperBase@CloudStrapper${CIRCLE_BUILD_NUM}@g" ~/magma-experimental/roles/vars/defaults.yaml 
+            cp ~/magma-experimental/ansible.cfg ~/.ansible.cfg
+            aws s3 cp s3://cloudstrapper-ec2-pems/mpboot101.pem ~/magma-experimental/files/
+            aws s3 cp s3://cloudstrapper-ec2-pems/mphost101.pem ~/magma-experimental/files/
+            chmod 600 ~/magma-experimental/files/*.pem
+            ansible-playbook aws-prerequisites.yaml -e "dirLocalInventory=~/magma-experimental/files"
```

Comment:
Should we put the ansible in different steps? 
1 - Prepare node 
2 - Ansible run

It will give us more readability

Reply:
this is what happens:
#!/bin/bash -eo pipefail
ansible-playbook aws-prerequisites.yaml -e "dirLocalInventory=~/magma-experimental/files"
ansible-playbook devops-provision.yaml -e "dirLocalInventory=~/magma-experimental/files"
ansible-playbook devops-configure.yaml -i ~/magma-experimental/files/common_instance_aws_ec2.yaml -e "devops=tag_Name_ec2MagmaDevopsCloudstrapper" -e "dirInventory=~/magma-experimental/files" -u ubuntu --skip-tags buildMagma,pubMagma,helm -e "dirLocalInventory=~/magma-experimental/files"
ansible-playbook devops-init.yaml  -e "dirLocalInventory=~/magma-experimental/files" -e "devOpsAmi=CloudStrapper${CIRCLE_BUILD_NUM}"

ERROR! the playbook: aws-prerequisites.yaml could not be found

Exited with code exit status 1
CircleCI received exit code 1